models/articles.model.js

```diff
@@ -15,5 +15,27 @@ exports.selectArticleById = (article_id) => {
             return article.rows[0];
         }
     })
+}
 
+exports.updateArticleVotes = (article_id, votes) => {
```

Comment:
In regards to my previous comment about an extra test I think the code you have here deals with it but it would be a good thing to test for just incase there's any holes in your logic :)